src/editor/Core/ConfigurationDefaults.re

```diff
@@ -30,7 +30,8 @@ let getDefaultConfigString = configName =>
   "workbench.editor.showTabs": true,
   "workbench.iconTheme": "vs-seti",
   "workbench.sideBar.visible": true,
-  "workbench.statusBar.visible": true
+  "workbench.statusBar.visible": true,
+  "explorer.indentSize": 2,
```

Comment:
Could we rename this setting to `"workbench.tree.indent"` please?

Two reasons I'm thinking:
- VSCode uses `workbench.tree.indent` - that would help us keep our configurations compatible
- We'll likely include other tree views - both as part of the extension work, and for other features (like, showing marks in the sidebar) - it would make sense for this indent setting to be generalized across those.

Sorry to ask you to rename it everywhere! But I think it would help keep people from having to update their configuration if we had to change it down the road 😄